docs/api/golang.mdx

```diff
@@ -0,0 +1,301 @@
+    "parent": {
+      "comm": "sh",
+      "pid": 42678
+    },
+    "pid": 53660,
+    "tid": 53660
+  },
+  "runtime": {
+    "containerId": "1b0ba0f6f7f7251a8dedbee0a836dc3edadeba7444cd56d6afe0989b6007bcd7",
+    "containerImageDigest": "sha256:2919d0172f7524b2d8df9e50066a682669e6d170ac0f6a49676d54358fe970b5",
+    "containerImageName": "busybox",
+    "containerName": "mycontainer",
+    "containerPid": 42678,
+    "containerStartedAt": 1737743296076978537,
+    "runtimeName": "docker"
+  },
+  "timestamp": "2025-01-24T13:37:35.312976962-05:00",
+  "timestamp_raw": 1737743855312976962
+}
+```
+
+As expected, only events from `mycontainer` are captured and the runtime leaf
+contains all information related to the container creating the events. Also, a
+new timestamp field with a human friendly version is present.
+
+Don't forget to remove the containers create above:
+
+```bash
+$ docker rm mycontainer foocontainer
+```
+
+### Data Source
+
+So far, we've been using the json formatter to convert the events produced by a
+Gadget to json. However, if you're only interested on getting specific fields,
+it can be more performant to access them directly using a Field Accessor instead
+of performing a full json marshaling and unmarshaling.
+
+import Datasource from '!!raw-loader!./_gadgets/datasource/main.go';
+
+<CodeBlock language="go">{Datasource}</CodeBlock>
+
+Executing this program will output the following:
+
+```bash
+$ go run -exec sudo .
+command systemd-journal (761) opened /run/log/journal/02962d410aef4be9b1be7d022e8be679/system.journal
+command systemd-journal (761) opened /run/log/journal/02962d410aef4be9b1be7d022e8be679/system.journal
+command tmux: server (12404) opened /proc/38122/cmdline
+command systemd-oomd (1245) opened /proc/meminfo
+command systemd-oomd (1245) opened /sys/fs/cgroup/user.slice/user-1001.slice/user@1001.service/memory.pressure
+command systemd-oomd (1245) opened /sys/fs/cgroup/user.slice/user-1001.slice/user@1001.service/memory.current
+command systemd-oomd (1245) opened /sys/fs/cgroup/user.slice/user-1001.slice/user@1001.service/memory.min
+command systemd-oomd (1245) opened /sys/fs/cgroup/user.slice/user-1001.slice/user@1001.service/memory.low
+command systemd-oomd (1245) opened /sys/fs/cgroup/user.slice/user-1001.slice/user@1001.service/memory.swap.current
+command systemd-oomd (1245) opened /sys/fs/cgroup/user.slice/user-1001.slice/user@1001.service/memory.stat
+command systemd-oomd (1245) opened /proc/meminfo
+command dbus-daemon (3811) opened /sys/kernel/security/apparmor/.access
+command tmux: server (12404) opened /proc/38122/cmdline
+command systemd-oomd (1245) opened /proc/meminfo
+command systemd-oomd (1245) opened /proc/meminfo
+command tmux: server (12404) opened /proc/38122/cmdline
+```
+
+### Running Gadgets remotely with the gRPC runtime
+
+All the examples above run the eBPF locally. This example shows how to control a
+remote `ig` instance by using the gRPC runtime.
```

Comment:
You could add some details:
- The client does not need connectivity to the OCI registry.
- The client does not need credentials to the OCI registry.

Is it correct btw?